wp-content/themes/darklight-site/front-page.php

```diff
@@ -3,15 +3,44 @@
 * Template Name: front page
 */
 get_header();
-
 $image = wp_get_attachment_image_src(get_post_thumbnail_id($post->ID), 'single-post-thumbnail')[0]
 
 ?>
-
+<div class="body">
+<div class= "container">
+  <div class= "row">
+    <div class= "col-6">
 <section class="banner">
   <img src="<?php echo $image; ?>">
 </section>
-<h1>hello</h1>
+</div>
+<div class= "col-6 head">
+  <h1>Darklight</h1>
+  <br>
```

Comment:
I wouldnt recommend using `<br>` or `<br />`, you could get the same using either margin-bottom on the h1 (or give the h1 a class name), or give your header section a line-height